.gitignore

```diff
@@ -13,6 +13,7 @@ db.sqlite3
 
 
 # Distribution / packaging
+.idea
```

Comment:
please rebase.
this change already merged in:
https://github.com/redhat-beyond/lets-meet/pull/50

Reply:
Done